### PR TITLE
feat: use implement addFavorites and removeFavorites

### PR DESCRIPTION
**Pull request by TzuHanLiang**
### DEVELOP
- [x] userContext
- [x] marketContext
- [x] TickerSelectorBox
- [x] CryptoCard
- [x] NavBar
- [x] NavBarMobile
- [x] OpenSubTab
- [x] OrderSection
- [x] WalletPannel
- [x] IWalletBalance
- [x] IResult
- [x] IListCFDResult
- [x] ICDFDetailsResult

### CHECK LIST
-  Feature summary: #184 #193 #206 #205
-  [Naming convention](coding-convention/naming-convention.md) verification:  **checked**
-  Coding style verification: **checked**
-  Any new libraries: 1
  - @cafeca/lunar  v0.4.4
-  Any new Class / Component: 4
  - IWalletBalance
  - IResult
  - IListCFDResult
  - ICDFDetailsResult
-  Any new loop: 0
-  Any new recursive: 0
-  Any high risk operation: 0
-  Any SQL operation: 0

### UML
- [TBDCD00001.md](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDCD00001.md)

### CODE REVIEW

**Review by Luphia — APPROVED** [review]
- no bug found

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -5,36 +5,35 @@ import type {AppProps} from 'next/app';
...
-          pauseOnHover
-          theme="dark"
-          limit={10}
-        />
-
-        {/* </UserProvider> */}
-        {/* </MarketProvider> */}
+        <UserProvider>
+          <MarketProvider>
+            <ViewportProvider>
```
Comment (arealclimber) [inline src/pages/_app.tsx:18]: - `<GlobalProvider>`

**Comment by Luphia** [comment]
- src/_app.tsx
  - remove themeContext (merge into globalContext)
  - add globalContext
- src/components/nav_bar/nav_bar.tsx use userCtx.isConnected
- src/components/nav_bar/nav_bar_mobile.tsx use userCtx.isConnected

**Comment by TzuHanLiang** [comment]
replaced by #212

**Pull request closed without merging**